### DISCUSSION TITLE
How to change design of "planning route line"?

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#18260] [Q&A]
Subject says it.. When planning a route, the route is shown as a very thin blue dotted line. Nearly not visible at all. How can I change the color/thickness of this line?

**Comment by DmitryAlexei** [comment]
@ThomasMeister123 Try to change map style. There are two colors in Plan-a-route for route line: Blue and Purple.
In some map styles (for example OsmAnd basic map style, Desert, Winter and Ski) route line is blue, in others it is purple.
you can also add your point to this *Nice to have* topic https://github.com/osmandapp/OsmAnd/issues/18182

**Reply by ThomasMeister123** [reply]
OK, but the line is to thin no matter which color :( Why not use the same line as in "normal" Navigation?

**Reply by DmitryAlexei** [reply]
Please add your comment to #18182
